### PR TITLE
Longer Thing

### PR DESCRIPTION
**Pull request by PrincessAkira**
- Added Undo for TPM and UAC
- Added TPM Check Skip
- Used the actual right values for GameDVR and FSO
- Added Long Paths

### CODE REVIEW

**Comment by PrincessAkira** [comment]
tldr i dont agree with the values from https://github.com/ChrisTitusTech/winutil/commit/a785c75bee99a2fc9d4e00a280293582e849960d

**Comment by d1payan** [comment]
> tldr i dont agree with the values from [a785c75](https://github.com/ChrisTitusTech/winutil/commit/a785c75bee99a2fc9d4e00a280293582e849960d)

You don't have to agree with my values. Just Put your all registry values outside if condition and inside if condition only put the new item code, like what I did. Otherwise, any of the registry tweaks will not work. 

Because the code is saying if the path (HKCU:\System\GameConfigStore) is NOT there then only apply these tweaks. But that is not what we want , right ?

So, pull the code first from latest changes then create a pull request with whatever value you want.

**Comment by PrincessAkira** [comment]
> > tldr i dont agree with the values from [a785c75](https://github.com/ChrisTitusTech/winutil/commit/a785c75bee99a2fc9d4e00a280293582e849960d)
> 
> You don't have to agree with my values. Just Put your all registry values outside if condition and inside if condition only put the new item code, like what I did. Otherwise, any of the registry tweaks will not work.
> 
> Because the code is saying if the path (HKCU:\System\GameConfigStore) is NOT there then only apply these tweaks. But that is not what we want , right ?
> 
> So, pull the code first from latest changes then create a pull request with whatever value you want.

i meant the one i used instead of the hex ones!!!
:D
everythings dandy ^-^
theres just other values one can use. should have worded better mb <3

**Comment by d1payan** [comment]
> > > tldr i dont agree with the values from [a785c75](https://github.com/ChrisTitusTech/winutil/commit/a785c75bee99a2fc9d4e00a280293582e849960d)
> > 
> > 
> > You don't have to agree with my values. Just Put your all registry values outside if condition and inside if condition only put the new item code, like what I did. Otherwise, any of the registry tweaks will not work.
> > Because the code is saying if the path (HKCU:\System\GameConfigStore) is NOT there then only apply these tweaks. But that is not what we want , right ?
> > So, pull the code first from latest changes then create a pull request with whatever value you want.
> 
> i meant the one i used instead of the hex ones!!! :D everythings dandy ^-^ theres just other values one can use. should have worded better mb <3 I wanted to pull yours but then it wouldnt let me PR mine and i would have needed to discard it full due to Github logic _grrr_

I am not saying your values are bad :D
But put it outside if loop or just remove the if loop because this path ((HKCU:\System\GameConfigStore) exists by default. No need to test it.

But you can still do it right ? Just do the changes what I told and then commit again. It will be automatically in the PR. Or, you can just sqash the new commit with previous commit.

**Comment by PrincessAkira** [comment]
> d1payan

Thats what i meant.
I wanted to pull your commit but Github wouldnt let me or it discards my Pushes :D
image

**Comment by PrincessAkira** [comment]
Just did it by hand now. God i hate github

**Review by d1payan — COMMENTED** [review]
Yes that is what I was trying to say.

**Comment by d1payan** [comment]
> Just did it by hand now. God i hate github

I am new to it. So, just learnt some things in these few days :D

**Comment by PrincessAkira** [comment]
Idk why it has merge errors. ima fix em somehow
